prody/dynamics/adaptive.py

```diff
@@ -369,71 +325,57 @@ def runAlternatingAdaptiveANM(structureA, structureB, n_steps, **kwargs):
+    ensB.setCoords(coordsB.copy())
+    ensB.setWeights(weights)
+    ensB.addCoordset(coordsB.copy())
+
+    while n < n_steps:
+        LOGGER.info('\nStarting cycle {0} with {1}'.format(n + 1, getTitle(structureA, 'structure A')))
+        n_modes = calcStep(coordsA, coordsB, n_modes, ensA, defvecs, rmsds,
+                           resetFmin=resetFmin, **kwargs)
+        n += 1
         resetFmin = False
-        self.anmListA.append(modesetA)
-        self.coordsA = new_coordsA
-        self.ensembleA.addCoordset(new_coordsA)
-        self.whichModesA.append(modesetA[modesCrossingFmin])
-        self.rmsds.append(rmsd)
-
-        LOGGER.debug('Total time so far is %.2f minutes' % (
-            (time.time() - LOGGER._times['_prody_runManySteps'])/60))
-
-        LOGGER.info('\nStarting cycle {0} with initial structure {1}'.format(self.numSteps+1,
-                                                                                self.titleA))
-        rmsd, new_coordsA, modesetA, modesCrossingFmin, self.dList, self.n_modes = _runStep(self.coordsB,
-                                                                                                       self.coordsA,
-                                                                                                       n_modes=self.n_modes,
-                                                                                                       resetFmin=resetFmin,
-                                                                                                       dList=self.dList,
-                                                                                                       **kwargs)
-        self.numSteps += 1
-        self.anmListB.append(modesetA)
-        self.coordsB = new_coordsA
-        self.ensembleB.addCoordset(new_coordsA)
-        self.whichModesB.append(modesetA[modesCrossingFmin])
-        self.rmsds.append(rmsd)
-
-        LOGGER.debug('Total time so far is %.2f minutes' % (
-            (time.time() - LOGGER._times['_prody_runManySteps'])/60))
-
-        converged = _checkConvergence(
-            self.rmsds, self.coordsA, self.coordsB)
-        if converged:
-            # That way the original object is back to normal
-            self.coordsA = self.coordsA_0
-            # That way the original object is back to normal
-            self.coordsB = self.coordsB_0
-            LOGGER.debug('Process completed in %.2f hours' % (
-                (time.time() - LOGGER._times['_prody_runManySteps'])/3600))
+
+        if n_modes == 0:
+            LOGGER.report('Alternating Adaptive ANM converged in %.2fs.', '_prody_calcAdaptiveANM')
             break
 
-    for coordset in self.ensembleA.getCoordsets():
-        self.ensemble.addCoordset(coordset)
-    for coordset in reversed(self.ensembleB.getCoordsets()):
-        self.ensemble.addCoordset(coordset)
+        LOGGER.info('\nStarting cycle {0} with structure {1}'.format(n+1, getTitle(structureB, 'structure B')))
```

Comment:
This should be above so you report starting cycle 1 (n=0) too. 

Might also be good to have this as debug not info.